### PR TITLE
[SLAC22-338] Refactor image-embed caption logic; remove caption from …

### PR DESCRIPTION
**Pull request by dstumberger**
…carousel view mode

### CODE REVIEW

File: config/sync/core.entity_view_display.paragraph.image_embed.carousel.yml [hunk 1]
```diff
@@ -25,13 +23,7 @@ content:
     third_party_settings: {  }
     weight: 0
     region: content
-  field_long_text:
```
Comment (kmonahan) [inline config/sync/core.entity_view_display.paragraph.image_embed.carousel.yml:28]: Why was this removed? Carousel images do need the ability to have captions. Is that being handled some other way?

Comment (dstumberger) [reply config/sync/core.entity_view_display.paragraph.image_embed.carousel.yml:28]: The caption for carousel images is passed into templating through the media item, and when the paragraph's caption overrides the media item, the preprocess hook assigns the paragraph caption into the media caption field.
